### ISSUE TITLE
Microsoft Teams incompatible

### ISSUE THREAD

**Issue by aberios** [getstation/desktop-app#222]
- Application version: 2.4.0
 - OS: _MacOS 14.0_

1. Try logging into MS Teams account from GoDaddy-managed service

**Screenshots**
<img width="410" alt="Screenshot 2023-09-28 at 6 06 30 PM" src="https://github.com/getstation/desktop-app/assets/37582/e23f99d4-bcfa-423b-84b4-c80c862f93c5">

**Comment by tomas4** [comment]
> I just get a blank white screen in the Linux client, Station AppImage v2.4.0.

Me too

**Comment by mikhailsirenko** [comment]
I'm having a blank screen as well but on Windows 10.

**Comment by aberios** [comment]
Still doesn't seem to be working.

https://github.com/getstation/desktop-app/assets/37582/cdbcf145-bd73-4f11-b79b-005cc2ba33a9

**Comment by tomas4** [comment]
On Linux I was able to successfully log-in with my University account. I do not know if it may be relevant, but I used the Sign-in options link, then selected corporate login (or something like it), selected domain and then authenticated from our domain login dialog. The version used:

Stationversion 2.5.0
Electron25.8.2

Chrome114.0.5735.289

Node18.15.0

v811.4.183.29-electron.0

**Comment by villecoder** [comment]
This got fixed with v2.5.0.

**Comment by aberios** [comment]
Doesn't look like it did.

<img width="1475" alt="Screenshot 2023-10-20 at 12 48 45 PM" src="https://github.com/getstation/desktop-app/assets/37582/c93f4776-544c-4b1e-a034-c8dd8e08c787">

**Comment by aberios** [comment]
@tomas4 using "Sign-in options" did not work.  I entered my organization's domain, and it redirected me to the same login page that has the issue.

**Comment by viktor44** [comment]
The first problem for me is to reproduce the bug.
My Microsoft account doesn't redirect me to GaDaddy. How can I create one with this login type?

**Comment by tomas4** [comment]
@aberios I tested on new install on my notebook. Indeed, a direct login without "Sign-in Options" works for me in 2.5.0 Appimage on Linux. I had exactly the same screen as others (unusual browser) in 2.4.0. So something was fixed, but not for everyone it seems. 
Adding video with a successful login (from a clean start, with adding services and so on). Sorry it is so long and erratic, I was a little distracted by my family. It is quickly edited. After the log-in the loading of Teams took a long time and I was looking into the tab controls, but did not click anything there and in the end it loaded the content.

https://github.com/getstation/desktop-app/assets/11938487/c4525b8a-0cac-4f76-ae7d-305d01591547
